heidrun/lc_json.rb

```diff
@@ -0,0 +1,119 @@
+build_uri = lambda do |node|
+  uri = RDF::URI(node.value)
+  uri.scheme = 'http' unless uri.scheme
```

Comment:
I'm diving in here w/out having read the whole mapping or the provider details, but I'm assuming that If they support scheme-relative URLs then they must support HTTPS, so I'd vote for making it "https".